serviced-controller/daemon.go

```diff
@@ -54,7 +54,7 @@ func main() {
 		cli.BoolFlag{"disable-metric-forwarding", "disable forwarding of metrics for this container"},
 		cli.StringFlag{"metric-forwarder-port", defaultMetricsForwarderPort, "the port the container processes send performance data to"},
 		cli.BoolTFlag{"logstash", "forward service logs via filebeat"},
-		cli.StringFlag{"logstash-settle-time", "0s", "time duration to wait for logstash to flush log messages before closing"},
+		cli.StringFlag{"logstash-settle-time", "60s", "time duration to wait for logstash to flush log messages before closing"},
```

Comment:
60 seconds seems excessive.  this delay is (IIRC) after the process we are killing exits, right?  logstash should be looping every 5 seconds, so 6-10 seems more reasonable, rather than delaying the stop of all services by one minute every time.

Reply:
Even when I use 30s timeout, it's not enough time to create `<container_id>.container.log` file in `/tmp/` folder. That file just missing in `/tmp/`.
When I use 60s timeout, container is shut down after 45 sec with appropriate `/tmp/<container_id>.container.log` file which contains next log messages:
```
...
...
...
2019/08/23 06:40:24.226740 tls.go:200: WARN SSL/TLS verifications disabled.
2019/08/23 06:40:39 200 45.446111ms POST /api/metrics/store
2019/08/23 06:40:54 200 12.284218ms POST /api/metrics/store
I0823 06:41:08.926689 00001 controller.go:729] Notifying subprocess of signal terminated for service vt60qw9kdrk7wm7bdxb7ibbw
I0823 06:41:08.927063 00001 controller.go:506] Sending signal terminated to pid 51 provided by PIDFile.
I0823 06:41:08.927398 00001 controller.go:716] Closing healthExit on signal terminated for "vt60qw9kdrk7wm7bdxb7ibbw"
I0823 06:41:08.927450 00001 controller.go:718] Closed healthExit on signal terminated for "vt60qw9kdrk7wm7bdxb7ibbw"
I0823 06:41:08.929347 00001 controller.go:731] Notification complete for signal terminated for service vt60qw9kdrk7wm7bdxb7ibbw
2019/08/23 06:41:08 200 5.521782ms POST /api/metrics/store
2019/08/23 06:41:09 200 10.451476ms POST /api/metrics/store
2019/08/23 06:41:24 200 15.050974ms POST /api/metrics/store
2019/08/23 06:41:39 200 7.980083ms POST /api/metrics/store
```

Reply:
https://github.com/control-center/serviced/pull/3755

Reply:
Again, the *.container.log files in /tmp are only created if we fail to shutdown cleanly.  What you showed above was a symptom of the fact that the process wasn't getting killed.  the logstash settle time is a delay imposed after the killed process exits to allow logstash to complete its tasks, and that should not take longer than 5-10 seconds.  leaving this at 60 will ensure that it takes 1 minute longer than needed to stop any service.